src/vs/workbench/contrib/notebook/browser/controller/executeActions.ts

```diff
@@ -109,6 +108,36 @@ async function runCell(editorGroupsService: IEditorGroupsService, context: INote
 	}
 }
 
+function handleAutoReveal(cell: ICellViewModel, notebookEditor: IActiveNotebookEditor, configurationService: IConfigurationService): void {
+	const revealPercent = configurationService.getValue<number | undefined>('notebook.scrolling.revealPercent');
+	const revealThreshold = configurationService.getValue<number | undefined>('notebook.scrolling.revealThreshold');
+	if (revealPercent && revealThreshold) { // "smart" reveal, only if both settings are set (for AI execution tracking primarily)
```

Comment:
`revealPercent && revealThreshold` won't work if once of them is `0`

Reply:
will check undefined, good catch